cllc-public-app/ClientApp/src/app/bceid-confirmation/bceid-confirmation.component.ts

```diff
@@ -79,7 +79,7 @@ export class BceidConfirmationComponent {
     let payload = JSON.stringify(account);
     this.busy = this.dynamicsDataService.createRecord('account', payload)
       .then((data) => {
-        window.location.reload();
+        this.currentUser.isNewUser = false;
       });
```

Comment:
Please set the other fields for currentUser based on the response from the account service, such as the accountid and contactid.